src/scss/_navbar.scss

```diff
@@ -0,0 +1,20 @@
+.navbar-brand {
+  font-size: .9rem;
+
+  @include media-breakpoint-up(sm) {
+    font-size: 1rem;
+  }
+
+  @include media-breakpoint-up(md) {
+    font-size: 1.25rem;
+  }
+
+  @include media-breakpoint-up(lg) {
+    font-size: .9rem;
+    margin-right: .25rem !important;
```

Comment:
.9rem gets a little tiny at this breakpoint, but we can revisit later since this would probably require a template part override to adjust the width of the h1/span around `.navbar-brand`